pixi.toml

```diff
@@ -0,0 +1,53 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+# -*- mode: toml -*-
+
+#                             VVVVVV  minimum `pixi` version
+"$schema" = "https://pixi.sh/v0.36.0/schema/manifest/schema.json"
+
+[project]
+name = "napari-feedstock"
+version = "3.51.0"  # conda-smithy version used to generate this file
```

Comment:
Will we need to bump it in the future?

Reply:
Nope, this is auto-generated and gets updated automatically on rerenders.